doc/Type/Mu.pod6

```diff
@@ -609,9 +609,10 @@ Returns the given item to the enclosing C<gather> block, without introducing a n
 
     method so()
 
-Returns a C<Bool> value representing the result of negating (with implicit
-conversion to C<Bool>) the expression and then again computing the negation of
-the result, thus computing the C<Bool> equivalent to the original expression.
+Evaluates the item in boolean context (and thus collapses Junctions),
+and returns the result.
+It is the opposite of C<not>, and equivalent to the
+L«C<?> operator|/language/operators#prefix_?».
```

Comment:
Careful when you say equivalent, because things like precedence might vary. Maybe add an example.

Reply:
Good point.